resources/backup-vaults-access-policies.go

```diff
@@ -117,3 +119,10 @@ func (b *BackupVaultAccessPolicy) Remove() error {
 func (b *BackupVaultAccessPolicy) String() string {
 	return b.backupVaultName
 }
+
+func (b *BackupVaultAccessPolicy) Filter() error {
+	if strings.HasPrefix(b.backupVaultName, "aws/efs/automatic-backup-vault") {
+		return nil
```

Comment:
Sorry just checked other resources. I do think we want the fmt.Error here

Reply:
no issue, thanks for quick review